recipe/meta.yaml

```diff
@@ -27,6 +27,13 @@ requirements:
     - defusedxml
     - future
     - packaging
+    {% if noarch_platform == "win" %}
+    - __win
+    - bottle
+    - requests
+    {% else %}
+    - __unix
+    {% endif %}
```

Comment:
Note the hash of the package is different for these two cases.

* [`noarch/glances-3.2.5-pyh08f2357_1.tar.bz2` (Windows)]( https://anaconda.org/conda-forge/glances/files/modal/info/62548379bfc43bf5079d2063 )
* [`noarch/glances-3.2.5-pyh2a2186d_1.tar.bz2` (Unix)]( https://anaconda.org/conda-forge/glances/files/modal/info/62547e3586b0cbb131ee497b )